README.md

```
@@ -6,7 +6,7 @@
 
 **Note:** This repository contains DevOps interview questions and answers. Please check the different sections for specific topics like Docker, Kubernetes, CI/CD, etc.
 
-### Table of Contents
+### Table of Contents from ajay
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)